Source/WebCore/workers/WorkerOrWorkletThread.cpp

```diff
@@ -63,7 +63,7 @@ static UniqueRef<WorkerRunLoop> constructRunLoop(WorkerThreadMode workerThreadMo
 }
 
 WorkerOrWorkletThread::WorkerOrWorkletThread(const String& inspectorIdentifier, WorkerThreadMode workerThreadMode)
-    : m_inspectorIdentifier(inspectorIdentifier)
+    : m_inspectorIdentifier(crossThreadCopy(inspectorIdentifier)) // inspectorIdentifier is accessed from the main thread.
```

Comment:
It seems `WorkerThread` and `AudioWorkletThread` already provide isolated copy of identifier to `WorkerOrWorkletThread`, is there any other case that does not provide isolated string?
If we are going to make this change, may be we can drop the isolated copy in the other places?